packages/clay-drop-down/src/index.tsx

```diff
@@ -5,8 +5,8 @@
  */
 
 import ClayDropDown, {Align} from './DropDown';
-import {ClayDropDownWithBasicItems} from './DropDownWithBasicItems';
+import {ClayDropDownWithBasicItems, IItem} from './DropDownWithBasicItems';
```

Comment:
Rather than exporting `IItem` its probably best to do something like `React.ComponentProps<typeof ClayDropDownWithBasicItems>['items'][number];`

I know its a bit longer, but I think its a little more clear. Especially because `IItem` comes from `DropDownWithBasicItems`.

What do you think?

Reply:
Hmm, I'm not sure if this could help because I need the IItem interface and if I get from `ClayDropDownWithBasicItems` it will come with `Array<IItem>` I just tried to extract `IItem` but couldn't do anything to get the result.

Reply:
Sorry, I totally missed this comment. You should be able to get `IItem` from `Array<IItem>` by adding `[number]`. This isn't vitally important, but it might be helpful to have consistency.